tests/common/gu_utils.py

```diff
@@ -33,7 +35,37 @@ def delete_tmpfile(duthost, tmpfile):
     duthost.file(path=tmpfile, state='absent')
 
 
-def apply_patch(duthost, json_data, dest_file):
+def format_json_patch_for_multiasic(duthost, json_data, is_asic_specific=False):
+    if is_asic_specific:
+        return json_data
+
+    json_patch = []
+    if duthost.is_multi_asic:
+        num_asic = duthost.facts.get('num_asic')
+
+        for operation in json_data:
+            path = operation["path"]
+            if path.startswith(HOST_NAME) and ASIC_PREFIX in path:
+                json_patch.append(operation)
+            else:
+                template = {
+                    "op": operation["op"],
+                    "path": "{}{}".format(HOST_NAME, path)
+                }
+
+                if operation["op"] in ["add", "replace", "test"]:
+                    template["value"] = operation["value"]
+                json_patch.append(template.copy())
+                for asic_index in range(num_asic):
+                    asic_ns = "{}{}".format(ASIC_PREFIX, asic_index)
+                    template["path"] = "{}{}".format(asic_ns, path)
+                    json_patch.append(template.copy())
+        json_data = json_patch
+
+    return json_data
+
+
+def apply_patch_wrapper(duthost, json_data, dest_file):
```

Comment:
>apply_patch_wrapper [](http://example.com/codeflow?start=4&length=19)

keep original function name since there is nothing changed in function body.

Reply:
Changed it back to the original name.